samples/cellular/modem_shell/src/link/link_shell.c

```diff
@@ -357,6 +358,40 @@ static const char link_propripsm_usage_str[] =
 	"  -e, --enable,       Enable proprietary PSM\n"
 	"  -h, --help,         Shows this help information";
 
+static const char link_uiccpowersave_usage_str[] =
+	"Usage: link uiccpowersave [options] | --read\n"
+	"Options:\n"
+	"  -r, --read,         Read and print the current UICC power saving mode.\n"
+	"      --default_mode, Default functionality.\n"
```

Comment:
Let's align the option names when it has been decided how the different modes are named.